src/CommitmentsV2/SFA.DAS.CommitmentsV2.ExternalHandlers/DependencyResolution/IoC.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.DependencyResolution;
+using SFA.DAS.CommitmentsV2.Shared.DependencyInjection;
+using SFA.DAS.PAS.Account.Api.ClientV2.DependencyResolution;
+using SFA.DAS.ReservationsV2.Api.Client.DependencyResolution;
+using SFA.DAS.UnitOfWork.EntityFrameworkCore.DependencyResolution.StructureMap;
+using SFA.DAS.UnitOfWork.NServiceBus.DependencyResolution.StructureMap;
+using StructureMap;
+using ApprenticeshipInfoServiceRegistry = SFA.DAS.CommitmentsV2.DependencyResolution.ApprenticeshipInfoServiceRegistry;
+using EncodingRegistry = SFA.DAS.CommitmentsV2.DependencyResolution.EncodingRegistry;
+
+namespace SFA.DAS.CommitmentsV2.ExternalHandlers.DependencyResolution
+{
+    public static class IoC
+    {
+        public static void Initialize(Registry registry)
+        {
+            registry.IncludeRegistry<ConfigurationRegistry>();
```

Comment:
I think a few of these Registry's are not needed now that we are only handling the one event. Off the top of my head:
- ApprenticeshipInfoServiceRegistry
- PasAccountApiClientRegistry
- EmployerAccountsRegistry
- ReservationsApiClientRegistry

Possibly others?

Reply:
Removed